main.go

```diff
@@ -25,7 +25,7 @@ func main() {
 		)
 		maxProjects  = cli.Option("-mp, --max-projects").Uint(1000)
 		fullProjects = cli.Option("-fp, --full-projects").Uint(3)
-		template     = cli.Option("-t, --template").Enum("one-page", "one-page", "full")
+		template     = cli.Option("-t, --template").Enum(TemplateOnePage, TemplateOnePage, TemplateFull)
```

Comment:
Please skip this change in this PR; as we discussed these are not really template names and we should improve it in renaming the flag altogether.

Reply:
I have pushed up new changes now so that this PR only fixes the bug that it was originally meant to fix!